pages/linux/ldconfig.md

```diff
@@ -0,0 +1,15 @@
+# ldconfig
+
+> Configure symlinks and cache for shared library dependencies.
+
+- Update symlinks and rebuild the cache. (Usually run when a new library is installed):
```

Comment:
Don't think the full stop is needed here. Also, I'm not sure that `Usually` needs a capital letter at the beginning.

Reply:
done